audit-base/src/main/java/com/wultra/core/audit/base/database/DatabaseAuditWriter.java

```diff
@@ -281,12 +287,15 @@ public void scheduledFlush() {
     }
 
     /**
-     * Scheduled cleanup of audit data in database.
+     * Scheduled cleanup of audit data in the database.
      */
-    @Scheduled(fixedDelayString = "${audit.cleanup.delay.fixed:3600000}", initialDelayString = "${powerauth.audit.cleanup.delay.initial:1000}")
+    @Scheduled(cron = "${audit.cleanup.cron:0 0 0/1 * * *}")
+    @SchedulerLock(name = "audit.cleanup", lockAtLeastFor = "${audit.cleanup.lockAtLeastFor:5s}", lockAtMostFor = "${audit.cleanup.lockAtMostFor:50m}")
     public void scheduledCleanup() {
-        logger.debug("Scheduled audit log cleanup called");
+        logger.info("action: scheduledCleanup, state: initiated");
+        LockAssert.assertLocked();
```

Comment:
Can we also add a log message when cleanup actually starts?

Reply:
I see, a good point!